README.md

```diff
@@ -1,7 +1,7 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_get_data_DOM/)
+    - [DEMO LINK](MaksymShevch.github.io/js_get_data_DOM/)
```

Comment:
Fix the demo link and update it in the PR description
```suggestion
    - [DEMO LINK](https://MaksymShevch.github.io/js_get_data_DOM/)
```

Reply:
thank you! i didn't understand why it didn't work